tests/assets/utils.ts

```diff
@@ -1,6 +1,9 @@
 import { instantMeiliSearch } from '../../src'
 import { MeiliSearch } from 'meilisearch'
 
+const HOST = 'http://localhost:7700'
```

Comment:
Cosmetic change? 💅

Reply:
Haha, yes I was annoyed that it was hardcoded bellow. My bad if this is not the right PR for this change